### ISSUE TITLE
Is there any way to figure out why certain components don't work?

### ISSUE THREAD

**Issue by drcmda** [xyc/vscode-mdx-preview#2]
For instance im trying this: https://codesandbox.io/s/rrppl0y8l4

test.js

```jsx
import * as THREE from 'three'
import React from 'react'
import { Canvas } from 'react-three-fiber'

function Thing({ vertices = [[-1, 0, 0], [0, 1, 0], [1, 0, 0], [0, -1, 0], [-1, 0, 0]] }) {
  return (
    <group ref={ref => console.log('we have access to the instance')}>
      <line>
        <geometry
          attach="geometry"
          vertices={vertices.map(v => new THREE.Vector3(...v))}
          onUpdate={self => (self.verticesNeedUpdate = true)}
        />
        <lineBasicMaterial attach="material" color="black" />
      </line>
      <mesh onClick={e => console.log('click')} onPointerOver={e => console.log('hover')} onPointerOut={e => console.log('unhover')}>
        <octahedronGeometry attach="geometry" />
        <meshBasicMaterial attach="material" color="peachpuff" opacity={0.5} transparent />
      </mesh>
    </group>
  )
}

export default function App() {
  return (
    <Canvas>
      <Thing />
    </Canvas>
  )
}
```

index.mdx

```
import App from './test'

# Test

<div style={{ position: 'relative', width: 300, height: 400, background: 'black' }}>
  <App />
</div>
```

It flashes up, then the preview page goes blank.

**Comment by xyc** [comment]
@drcmda thanks for reporting this issue!

Generally the error will surface in react-error-overlay, but if it's not, you can open "Developer: Open Webview Developer Tools" (from command palette) to inspect the error.

Looks that the specific error with your file is `TypeError: Cannot set property 'getCurrentStack' of undefined`. After some google searches, I found it to be similar to https://github.com/diegomura/react-pdf/issues/336. And the root cause is:
> react-reconciler appears to be blowing up if it’s not set to production

I'll fix that by default providing NODE_ENV="production" to the globals in the next release. Please stay tuned!

**Comment by xyc** [comment]
@drcmda Fixed in 0.1.3. If you update the extension you should be able to get the fix.
FYI, you can just preview `index.js` instead of `.mdx` in your example: (if it's renders to #root)
<img width="972" alt="Screen Shot 2019-04-08 at 8 46 46 PM" src="https://user-images.githubusercontent.com/679275/55772703-32e94300-5a42-11e9-83d8-625ff240758b.png">

for the time being the DOM structure of the preview webview is like this:
```html
<div id="vscode-mdx-preview_root">
    <div id="root">
    </div>
</div>
```

**Comment by drcmda** [comment]
Wow, this is insane! Thank you so much! This will eradicate the need for slides. 😍
